src/Main/PlaneSliders.jsx

```diff
@@ -1,17 +1,24 @@
 import React, { useEffect, useRef, useState } from 'react'
-import { useActor } from '@xstate/react'
-import { Chip, Icon, IconButton, Slider, Tooltip } from '@mui/material'
+import { useSelector } from '@xstate/react'
 import {
   visibleIconDataUri,
   invisibleIconDataUri,
   pauseIconDataUri,
   playIconDataUri
 } from 'itk-viewer-icons'
+import Button from 'react-bootstrap/Button'
+import Form from 'react-bootstrap/Form'
+import Image from 'react-bootstrap/Image'
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger'
+import Tooltip from 'react-bootstrap/Tooltip'
+import cn from 'classnames'
+import Badge from 'react-bootstrap/Badge'
 import '../style.css'
 
-function PlaneSliders(props) {
+const PlaneSliders = React.memo(function PlaneSliders(props) {
   const { service } = props
-  const [state, send] = useActor(service)
+  const state = useSelector(service, (state) => state)
```

Comment:
We can have more than 1 useSelector to isolate the state this component is interested in.